tests/46direct/01directmessage.pl

```diff
@@ -0,0 +1,143 @@
+use Future::Utils qw( repeat );
+
+my $next_device_message_txn_id = 0;
+
+push our @EXPORT, qw( matrix_send_to_device_message );
+
+sub matrix_send_to_device_message
```

Comment:
A bold choice of name there. Maybe `send_message_to_device`; or perhaps is the "message" part implied? Could it be just `send_to_device` ?